refaster-support/src/test/java/tech/picnic/errorprone/refaster/ErrorProneForkTest.java

```diff
@@ -0,0 +1,65 @@
+package tech.picnic.errorprone.refaster;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.ERROR;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.CompilationTestHelper;
+import com.google.errorprone.ErrorProneOptions;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.ClassTreeMatcher;
+import com.google.errorprone.matchers.Description;
+import com.sun.source.tree.ClassTree;
+import org.junit.jupiter.api.Test;
+
+final class ErrorProneForkTest {
+  @Test
+  void isErrorProneForkAvailable() {
+    assertThat(ErrorProneFork.isErrorProneForkAvailable())
+        .isEqualTo(Boolean.TRUE.toString().equals(System.getProperty("error-prone-fork-in-use")));
+  }
+
+  @Test
+  void isSuggestionsAsWarningsEnabledWithoutFlag() {
+    CompilationTestHelper.newInstance(TestChecker.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "// BUG: Diagnostic contains: Suggestions as warnings enabled: false",
+            "class A {}")
+        .doTest();
+  }
+
+  @Test
+  void isSuggestionsAsWarningsEnabledWithFlag() {
+    assumeTrue(ErrorProneFork.isErrorProneForkAvailable());
```

Comment:
TIL, very nice i.c.w. the `error-prone-fork` profile configuration. This is one thing I want to run in an IDE though to see 😄 Will try to do so tomorrow.